name.abuchen.portfolio.ui/src/name/abuchen/portfolio/ui/messages.properties

```diff
@@ -997,6 +997,13 @@ LabelEarningsPerYear = Year
 
 LabelEarningsSelectStartYear = Earnings starting with year:
 
+LabelEarningsExpenses = Earnings \u0026 Expenses
+
+LabelExpenses = Expenses
+
+LabelExpenses = Ausgaben
```

Comment:
Der Eintrag ist doppelt - einmal Deutsch, einmal Englisch. Es ist Zufall welcher Wert genommen wird.

Ich verwende zum Editieren der Message Files den ResourceBundleEditor. Das ist ein Plug-in zu Eclipse. Damit hat man ein schönes UI. Solche Fehler können nicht passieren. Und es ist auch direkt richtig formatiert. Und die Umlaute richtig escaped, etc.